packages/driver/cypress/e2e/e2e/rerun.cy.js

```diff
@@ -8,6 +8,36 @@ if (window.top.runCount == null) {
   window.top.runCount = 0
 }
 
+// This spec in the driver has some weird reloading that occurs. It triggers a simulation of 
+// essentially reloading the page in run mode and that effectively wipes away some state that we 
+// need for protocol. Rather than trying to handle this edge case in a complicated one-off fashion, 
+// just ensure that the protocol events fire for this one spec.
+const getCypressProtocolElement = () => {
```

Comment:
Can you link to where this code is copied from.

Reply:
Updated: [`abdc741` (#29183)](https://github.com/cypress-io/cypress/pull/29183/commits/abdc741c92abe51f30c7739fb2cf762e474a471c)